global/scripts/CorrThick.sh

```diff
@@ -0,0 +1,181 @@
+#!/bin/bash
+set -eu
+
+pipedirguessed=0
+if [[ ""$HCPPIPEDIR:-"" == "" ]]
+then
+	pipedirguessed=1
+	#fix this if the script is more than one level below HCPPIPEDIR
+	export HCPPIPEDIR="$(dirname -- "$0")/.."
+fi
+
+source "$HCPPIPEDIR/global/scripts/newopts.shlib" "$@"
+source "$HCPPIPEDIR/global/scripts/debug.shlib" "$@"
+
+#description of script/command
+opts_SetScriptDescription "Run curvature-corrected (folding-compensated) cortical thickness python script and save curvature-corrected (folding-compensated) thickness, curvatures, regression coefficients, and resampled outputs.
+
+References:
+  - https://www.biorxiv.org/content/10.1101/2025.05.03.651968v1
+  - https://onlinelibrary.wiley.com/doi/full/10.1002/hbm.25776
+Please cite these works when using this measure.
+
+Requirements:
+  Python dependencies: numpy, nibabel, scipy, psutil
+  Standard libraries: os, math, multiprocessing, concurrent.futures
+
+Running stand-alone:
+  To run outside the full HCP pipeline, prepare a batch script that defines (at least):
+    - Subject directory
+    - Subject list
+    - Location of the HCP Pipeline environment setup script (SetUpHCPPipeline.sh)
```

Comment:
Subject directory and subject are arguments, defining them in the batch is merely for convenience in order to pass them to the script.  The script will already complain if they didn't source a `SetUp...` script, as `HCPPIPEDIR` won't be set, and `pipedirguessed` will be set, and an error will be generated if it gets through `opts_parseArguments` without other issues.  We can probably assume the user already ran at least PostFreeSurfer (potentially with custom arguments) if they are looking at the help info of this script, and knows the general idea of setting up to run our scripts.